frontend/src/components/CardGridEvent.jsx

```diff
@@ -0,0 +1,23 @@
+import CardEvent from './CardEvent'
+
+const CardGridEvent = (data) => {
+    let eventData = []
+    if (data.data != undefined) {
+        eventData = data.data
+    }
```

Comment:
Since this is a UI component, avoid introducing data logic. The data props here should be a formatted data object. I would move the data.data (which is specific for your axios calls), to the upper layer.

Reply:
Done